test/TestSetup.sol

```diff
@@ -127,6 +128,43 @@ contract TestSetup is Test, Configured {
         avatar.enableModule(module);
     }
 
+    function _executeTestTransaction(string memory filename) internal {
+        // This is so we can just call execTransactionFromModule to simulate executing transactions without signatures.
+        _addModule(IAvatar(morphoDao), address(this));
```

Comment:
I'm worried about how this will behave if there are multiple test transactions to execute. Maybe we need to add a check to make sure that address(this) is not already a module, or move this step to the setUp().